examples/flexvolume/nginx.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    volumeMounts:
+    - name: test
+      mountPath: /data
+    ports:
+    - containerPort: 80
+  volumes:
+  - name: test
+    flexVolume:
+      driver: "kubernetes.io/lvm"
+      fsType: "ext4"
+      options:
+        volumeID: "vol1"
+        size: "1000m"
+        volumegroup: "kube_vg"
+        fsType: "ext4"
```

Comment:
I thought about it, the FS type argument is more relevant to the driver and will not require an extra argument to the call out.
If it makes it more intuitive to have it at the same level as options I can add it and add fstype as an argument to the callout.